.eslintrc.js

```diff
@@ -1,8 +1,28 @@
 module.exports = {
-  root: true,
-  extends: '@react-native-community',
-  extends: 'standard',
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
+    'plugin:prettier/recommended', // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+  ],
+  plugins: ['@typescript-eslint'],
   rules: {
-    semi: [1, 'never'],
+    // Type is enforced by callers. Not entirely, but it's good enough.
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    // Aries protocol defines attributes with snake case.
+    '@typescript-eslint/camelcase': 'off',
+    '@typescript-eslint/no-use-before-define': ['error', { functions: false, classes: false, variables: true }],
+    '@typescript-eslint/explicit-member-accessibility': 'error',
+    'no-console': 'error',
+    // Because of early development, we only warn on ts-ignore. In future we want to move to error
+    '@typescript-eslint/ban-ts-comment': 'warn',
   },
+  overrides: [
+    {
+      files: ['*.test.*'],
+      rules: {
+        'no-console': 'off',
```

Comment:
I am curious @TimoGlastra if you'd suggest in the future the removal of all console logs--or if given our context it's a non-issue or given our current stage of development it's a shorter-term rule.

Reply:
Yeah I think with the recent logger additions in AFJ it would be nice to use that for everything. We removed all console.logs in AFJ now and instead use the logger interface. This makes it a lot more customizable. AFJ has a default ConsoleLogger that we can start with, however a more advanced logger would be nice to add over time

Reply:
Agreed--your work on the logger additions in AFJ are quite cool + appreciated. I also remembered as of right now that the babel config is removing console logs on a production build--curious if we should change this behavior or when it makes sense to alter it.